static/prompts/image-alt-text.prompt

```diff
@@ -16,16 +16,15 @@ You are an expert SEO consultant, and your goal is to suggest a description for
 11. End the alt text sentence with a period.
 12. For infographics, describe the key data points and trends.
 
-
-
 ### Response Format:
 Your response must be a valid JSON object using the following structure:
 [{
   "image_url": "string",
   "suggestion": "string",
   "ai_rationale": "string",
   "confidence_score": number,
-  "is_appropriate": boolean
+  "is_appropriate": boolean,
+  "language": "string"
```

Comment:
Just FYI, since we need to change the prompt, we will need to consider this in the AI Ethics review / report generator so what they approve is the same that we use in the audit.